man/compose_data.Rd

```diff
@@ -61,19 +62,14 @@ If `TRUE` and `m_0` and `S_0` are specified, `m_0` and `S_0` will also be scaled
 components necessary to explain at least the pca.cutoff of the total variance. (default: .95) Ignored if
 `pca.observation = FALSE`. (default: 1)}
 
-\item{m_0, S_0}{Optionally, prior means (m_0) and/or prior scatter matrices (S_0) for all categories. Each should be
-a list, with each element being the mean/scatter matrix for a specific category. Elements should be ordered in the same order as
-the levels of the category variable in \code{exposure} and \code{test}. The means and scatter matrices could be
-estimated, for example, from phonetically annotated speech recordings (see \code{\link{make_NIW_prior_from_data}}
-for a convenient way to do so). To aspects should be kept in mind, however. First, an \emph{inferred} scatter
-matrix includes variability from internal perceptual and/or external environmental noise, \emph{in addition} to the motor noise that
-is reflected in production data. That means that any prior for the scatter matrix (S_0) should take into account
-not only the variability in the production data but also any additional variability added by internal (perceptual)
-or external (environmental) noise. Second, the prior scatter matrix has an implicit nu associated with it, so that
-the nu inferred by \code{\link{infer_prior_beliefs}} is best thought of as a multiple of the implicit nu used
-during the creation of the scatter matrix S_0. For that reason, we recommend the use of nu = D + 2 in the call to
-\code{\link{make_NIW_prior_from_data}} (the default), since the S_0 obtained that way is identical to the category
-covariance matrix Sigma.}
+\item{lapse_rate, mu_0, Sigma_0}{Optionally, lapse rate, prior expected category means (mu_0) and/or prior expected
+category covariance matrices (Sigma_0) for all categories. Lapse rate should be a number between 0 and 1. For mu_0
+and Sigma_0, each should be a list, with each element being the expected mean/covariance matrix for a specific
+category prior to updating. Elements of mu_0 and Sigma_0 should be ordered in the same order as the levels of the
+category variable in \code{exposure} and \code{test}. These prior expected means and covariance matrices could be
+estimated, for example, from phonetically annotated speech recordings (see \code{\link{make_MVG_from_data}}
+for a convenient way to do so). Internally, m_0 is then set to mu_0 (so that the expected value of the prior
+distribution of means is mu_0) and S_0 is set so that the expected value of the inverse-Wishart is Sigma_0 given nu_0.}
```

Comment:
did you mean to write `S_0 `is set to `Sigma_0`?

Reply:
Thanks for asking. No, S_0 is not set to Sigma_0. That wouldn't work. S_0 ... S_n are *scatter* matrices (a matrix of uncentered sums of squares; like sum of squares matrices used in an ANOVA but they are centered). The elements of such a matrix go up with each observation. To get Sigma_n from S_n, one needs to 'normalize' S_n with nu. What the stan code does is to force S_0 to Sigma_0 such that Sigma_0 is the expected value of the IW distribution with scatter matrix S_0 and degrees of freedom nu_0.